airtime_mvc/application/views/scripts/login/index.phtml

```diff
@@ -1,12 +1,7 @@
 <div class="login_box">
-    <div class="logobox">&nbsp;</div>
-        <h2><?php echo _("Login") ?></h2>
+    <div class="logobox"></div>
+        <h2>Login</h2>
     <div id="login" class="login-content clearfix">
-        <?php if(isset($this->demo) && $this->demo == 1){?>
```

Comment:
It feels right to remove the demo code. We should remove the correponding code in `LoginController` as well.
https://github.com/LibreTime/libretime/blob/6c4b1b5e71f8de260a1b04ee3f04c7489b1832bd/airtime_mvc/application/controllers/LoginController.php#L100-L102